utils/daysSinceDate.js

```diff
@@ -0,0 +1,9 @@
+export default function daysSinceDate(date) {
+  if (!date) {
+    return 100;
```

Comment:
Why 100?

Reply:
Was an old statement just statement i can delete now for sure! Thanks!